src/controllers/audiomessage.controller.ts

```diff
@@ -1,278 +1,195 @@
- * @access Public
- */
-export const uploadMusicFiles = async (req, res) => {
-  // Extract relevant data from the request body
-  const { Musictitle, artist, lyrics } = req.body;
-
-  // Extract music and banner files from the request
-  const Music = req.files.Music[0];
+export const uploade = async (req,res) =>{
+  const {folderName, description } = req.body;
   const Banner = req.files.Banner[0];
+  console.log("banner:",Banner);
+  console.log(folderName);
+
+  const existingFolder = await FirstFolder.findOne({
+    folderName: { $regex: new RegExp(`^${folderName}$`, 'i') },
+  });
 
-  // Process music details
-  const AudioName = req.files.Music[0].originalname;
-  const MusicName = sanitizeFileName(AudioName);
-  const MusicKey = `${uuidv4()}-${MusicName}`;
+    if (existingFolder) {
+      return res.status(400).json({ message:"this Folder name already exists please change Folder name"});
+    }
 
-  // Process banner details
-  const file2Name = req.files.Banner[0].originalname;
+  const file2Name = Banner.originalname;
+    console.log(file2Name)
   const BannerName = sanitizeFileName(file2Name);
+  
   const Bannerkey = `${uuidv4()}-${BannerName}`
+  console.log(Bannerkey)
 
-  try {
-    // Set up parameters for uploading music file to S3
-    const params1 = {
-      Bucket: AWS_BUCKET_NAME,
-      Key: `uploads/${MusicKey}`,
-      Body: Music.buffer,
-      ContentType: Music.mimetype,
-    };
-
-    // Set up parameters for uploading banner file to S3
-    const params2 = {
+  try{
+    const params = {
       Bucket: AWS_BUCKET_NAME,
       Key: `uploads/${Bannerkey}`,
       Body: Banner.buffer,
       ContentType: Banner.mimetype,
-    };
-
-    // Create S3 upload commands for music and banner files
-    const command1 = new PutObjectCommand(params1);
-    const command2 = new PutObjectCommand(params2);
-
-    // Upload music file to S3
-    const uploaded1 = await s3client.send(command1);
-
-    // Upload banner file to S3
-    const uploaded2 = await s3client.send(command2);
-
-    // Format lyrics (replace newline characters with '<br>')
-    const formattedLyrics = lyrics.replace(/\n/g, "<br>");
-
-    // Create a new audio entry in the database
-    const audio = await Audio.create({
-      Musictitle,
-      artist,
-      lyrics: formattedLyrics,
-      MusicKey: params1.Key,
-      BannerKey: params2.Key,
-      Audio_location: `https://${AWS_BUCKET_NAME}.s3.${AWS_REGION}.amazonaws.com/${params1.Key}`,
-      Banner_location: `https://${AWS_BUCKET_NAME}.s3.${AWS_REGION}.amazonaws.com/${params2.Key}`,
-      download_file: `https://${AWS_BUCKET_NAME}.s3.${AWS_REGION}.amazonaws.com/${params1.Key}`,
-    });
+      };
+      console.log(params.Key)
 
-    // Respond with success message and the created audio entry
-    return res.status(201).json({
-      success: "song added successfully",
-      audio,
+      // Create a new command to upload the banner file to S3
+    const command = new PutObjectCommand(params);
```

Comment:
you are making a pu object but you are sending to s3 bucket , please refer to other code